Integrations/integration-jira.yml

```diff
@@ -380,67 +437,39 @@ script:
-            if (args.issueTypeName && args.issueTypeName.length > 0 ){
-                issue.fields.issuetype.name = args.issueTypeName;
-            }
-            if (args.issueTypeId && args.issueTypeId.length > 0 ){
-                issue.fields.issuetype.id = args.issueTypeId;
-            }
-            if (args.description && args.description.length > 0 ){
-                issue.fields.description = args.description;
-            }
-            if (args.labels && args.labels.length > 0 ){
-                issue.fields.labels = args.labels.split(",");
-            }
-            if (args.priorityName && args.priorityName.length > 0 ){
-                if (!issue.fields.priority) {
-                    issue.fields.priority = {};
-                }
-                issue.fields.priority.name = args.priorityName;
-            }
-            if (args.dueDate && args.dueDate.length > 0 ){
-                issue.fields.duedate = args.dueDate;
-            }
-            if (args.assignee && args.assignee.length > 0 ){
-                if (!issue.fields.assignee) {
-                    issue.fields.assignee = {};
-                }
-                issue.fields.assignee.name = args.assignee;
-            }
-            if (args.reporter && args.reporter.length > 0 ){
-                if (!issue.fields.reporter) {
-                    issue.fields.reporter = {};
-                }
-                issue.fields.reporter.name = args.reporter;
-            }
+            var url = 'rest/api/2/issue';
+            var issue = getIssueFields()
             var result = sendRequest('POST', url, JSON.stringify(issue));
             var jRes = JSON.parse(result.Body);
             var mdAndContext = generateMdContextCreateIssue(jRes);
             var md = tblToMd(command, mdAndContext['md'], "");
             return {Type: entryTypes.note, Contents: jRes, ContentsFormat: formats.json, HumanReadable: md, ReadableContentsFormat: formats.markdown, EntryContext: mdAndContext['context']};
+        case 'jira-edit-issue':
+            var url = 'rest/api/2/issue/' + String(args.issueId);
+            var issue = getIssueFields()
+            var result = sendRequest('PUT', url, JSON.stringify(issue));
+            if (result.StatusCode == 204) {
+                return 'Issue #' + String(args.issueId) + ' was updated successfully';
+            }
+            else return result;
+        case 'jira-get-status-ids':
+            var url = '/rest/api/2/issue/' + String(args.issueId) + '/transitions'
+            var result = sendRequest('GET', url);
+            var jRes = JSON.parse(result.Body);
+            var md = {}
+            for (i = 0; i < jRes.transitions.length; i++ ) {
+                md[jRes.transitions[i].name] = jRes.transitions[i].id;
+            }
+            md = objToMd(md);
+            return {Type: entryTypes.note, Contents: jRes, ContentsFormat: formats.json, HumanReadable: md, ReadableContentsFormat: formats.markdown};
```

Comment:
change all
`{Type: entryTypes.note, Contents: jRes, ContentsFormat: formats.json, HumanReadable: md, ReadableContentsFormat: formats.markdown}`
to 
```
{
    Type: entryTypes.note, 
    Contents: jRes, 
    ContentsFormat: formats.json, 
    HumanReadable: md, 
    ReadableContentsFormat: formats.markdown
}
```